agent/config/builder.go

```diff
@@ -828,6 +828,7 @@ func (b *Builder) Build() (rt RuntimeConfig, err error) {
 		EncryptVerifyOutgoing:            b.boolVal(c.EncryptVerifyOutgoing),
 		GRPCPort:                         grpcPort,
 		GRPCAddrs:                        grpcAddrs,
+		IgnoreBadCheckFiles:              b.boolVal(c.IgnoreBadCheckFiles),
```

Comment:
I like this name as a variable, but was thinking perhaps we could incorporate some more clarity by having something like "InvalidCheckFiles". That way if your check files are invalid, you set it to true! What do you think about this?

Reply:
I'm not sure I understand this one. My idea here is for SRE/DevOps/Infra folks to set this up in their Consul config so that there is a safeguard against bad checks causing Consul to not start.

So when reading this config param, I feel it clearly indicates that Consul should ignore bad/invalid check files (when true). When false, it doesn't ignore them and so Consul would behave in a default manner. So are you saying I should rename the param?

If not ☝️ , are you suggesting just the attr name to be changed? In that case, I was thinking that reading this variable/attr name in the code would still make it clear to the reader/maintainer, what the var is intending.

So having something like `InvalidCheckFiles` be set to true may only indicate that there are invalid files but not necessarily what should be done when they're encountered.

Does the distinction make sense?
Maybe I didn't understand you correctly. Please clarify.

Reply:
So, to be *completely* honest I think what I am suggesting the attr name be changed, but I'm not completely against the entire attr name I just dislike the usage of `Bad`.  It seems a tad vague and out of place. `OmitInvalidCheckFailes` or even `IgnoreInvalidCheckFiles` is fine with me, they're both _longer_ but I think a tad more clear.  
Honestly, if it's too much work to change the name, it's no big deal just ignore this comment.

Reply:
Oh definitely. (It wasn't clear to me from the original comment.) I think `Invalid` instead of `Bad` is better. At the time of the PR I wasn't sure if we had a formal definition for `invalid`. That said, I should be able to churn out a change real quick.